packages/plugin-eslint/src/bin.ts

```diff
@@ -1,3 +1,7 @@
+import process from 'node:process';
+import { Parser } from 'yargs/helpers';
 import { executeRunner } from './lib/runner/index.js';
 
-await executeRunner();
+const { runnerConfigPath, runnerOutputPath } = Parser(process.argv);
```

Comment:
Why don't we declare the required Yargs options? Then the args will actually be validated, and we'll get the inferred types.

The logic could live in `@code-pushup/utils`, since many plugins are using the same pattern.